### PR TITLE
Milestone 1- Project Setup

### PR DESCRIPTION
**Pull request by george-shammar**
This Pull Request seeks to merge the changes arising from the project set up to the default branch.  Below are the new features to be merged:

1. A new react app created with full features of the "create-react-app" tool- chain.
2. Prop-types added with npm.
3. Big.js added with npm. 
4. Deployment to heroku with create-react-app-buildpack.

### CODE REVIEW

**Review by taiwo2 — CHANGES_REQUESTED** [review]
# STATUS: INVALID 🔴 
@george-shammar well-done, Your project was invalidated due to a lack of descriptive PR. make it descriptive to tell what the milestone project meant to do.

File: src/App.css [hunk 1]
```diff
@@ -32,6 +32,7 @@
   from {
     transform: rotate(0deg);
   }
```
Comment (bellom) [inline src/App.css:34]: I think you should delete this file and any other unnecessary files, as well as the React svg Logo.
image

**Review by bellom — CHANGES_REQUESTED** [review]
## Hi @george-shammar  :clap: 

### Awesome job so far!! :+1: 

#### You have done wonderfully well, you passed `almost` all the checklist for this project :+1: just a few issues left to work on............ :slightly_smiling_face: 

# Status: Changes Required! ♻️ 

**1. Kindly update your `README` file**
**Let's make your repo look more professional by updating your `README` file**.
Checkout out the below link for help, feel free to follow this [Microverse template](https://github.com/microverseinc/readme-template) from pathwright

**2. Kindly check out my comments below;**

### ♻️ Kindly re-submit for review after making all those changes :slightly_smiling_face: Thumps -up await you.....

**Best**

File: README.md [hunk 2]
```diff
@@ -1,70 +1,71 @@
-# Getting Started with Create React App
+# math-calc-app
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+This is a simple calculator application developed to demonstrate React.js's functionality and for educational purpose.
```
Comment (rindrajosia) [inline README.md:3]: This is a simple calculator application developed to demonstrate React.js's functionality and for educational `purposes.`

**Review by rindrajosia — CHANGES_REQUESTED** [review]
Changes Required :red_circle:
To Highlight: :angel:

- Readme updated :+1: 

To improve :factory_worker: [REQUIRED]:

- [ ] Please fix the MIT link, and use tools like Grammarly to help you with your readme as it is the first thing users check.
- [ ] It will be nice if you remove all unnecessary files like: App.test.js, reportWebVitals.js, and setupTests.js. 
([screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone1/files-removed.png) from [pathwright](https://microverse.pathwright.com/library/fast-track-curriculum/69047/path/step/44896085/))

Nice to have [OPTIONAL] :

- [ ] As you are using ES6 syntax, please use the arrow function instead of the function keyword.
- [ ] It will be nice if you update the live demo based on the modifications that you made.

That's it, you can submit another code review request after making these changes :recycle: :innocent:
With all my respect :pray:
Rindra Josia

File: README.md (hunk 2 above)
Comment (george-shammar) [reply README.md:3]: @rindrajosia, This comment is not clear. I'm not sure what is required.

Comment (rindrajosia) [reply README.md:3]: Instead of Purpose, it should be Purposes

Comment (george-shammar) [reply README.md]: Okay. :smiley:

**Review by enaburekhan — APPROVED** [review]
# Status: Approve:heavy_check_mark:
**Hi George!
Congratulations:clap::clap:. Thank you for implementing the changes requested by the last reviewer.
Move on with the next milestone:walking:.**

Do have questions/comments? kindly drop them :point_down: and ping me on slack @ Eric Enaburekhan to respond.
Happy hacking:blush:
Eric Enaburekhan.

**Pull request merged**